routers/user/home.go

```diff
@@ -215,63 +219,48 @@ func Issues(ctx *context.Context) {
 		repos = ctxUser.Repos
 	}
 
-	allCount := 0
-	repoIDs := make([]int64, 0, len(repos))
-	showRepos := make([]*models.Repository, 0, len(repos))
 	for _, repo := range repos {
 		if (isPullList && repo.NumPulls == 0) ||
-			(!isPullList &&
-				(!repo.EnableIssues || repo.EnableExternalTracker || repo.NumIssues == 0)) {
+		(!isPullList &&
+		(!repo.EnableIssues || repo.EnableExternalTracker || repo.NumIssues == 0)) {
```

Comment:
I'm sorry, I don't quite follow. What do you mean by `go fmt`? The `go` formatting package?